File: static/code/sample/tethered/gdl-acservop8-b.py
# In this sample:
# Continuously set servo 1 from 0 to 180 degree

import time
from DUELink.DUELinkController import DUELinkController

availablePort = DUELinkController.GetConnectionPort()
duelink = DUELinkController(availablePort)

# method
def Set(servo, degree):
    duelink.Engine.ExecuteCommand(f"ServoSt({servo},{degree})")

while True:
    Set(1, 0)
    time.sleep(1)
    Set(1, 45)
    time.sleep(1)
    Set(1, 90)
    time.sleep(1)
    Set(1, 135)
    time.sleep(1)
    Set(1, 180)
    time.sleep(1)

    time.sleep(1)

File: static/code/sample/tethered/gdl-acservop8-b.cs
// In this sample:
// Continuously set servo 1 from 0 to 180 degree
using GHIElectronics.DUELink;

var availablePort = DUELinkController.GetConnectionPort();
var duelink = new DUELinkController(availablePort);

void Set(int servo, int degree) {    
    duelink.Engine.ExecuteCommand($"ServoSt({servo},{degree})");
}

while (true) {
    Set(1, 0);
    Thread.Sleep(1000);
    Set(1, 45);
    Thread.Sleep(1000);
    Set(1, 90);
    Thread.Sleep(1000);
    Set(1, 135);
    Thread.Sleep(1000);
    Set(1, 180);
    Thread.Sleep(1000);
    
    Thread.Sleep(1000);
}
